### ISSUE TITLE
403 Forbidden  nginx/1.4.6 (Ubuntu)

### ISSUE THREAD

**Issue by juancresc** [GMOD/docker-compose-galaxy-annotation#3]
I'm getting a 403 when accessing ip:8080

``` console
$ docker ps -a
CONTAINER ID        IMAGE                                      COMMAND                  CREATED             STATUS                       PORTS                                NAMES
af032f004c0c        nginx                                      "nginx -g 'daemon off"   9 minutes ago       Up 9 minutes                 443/tcp, 0.0.0.0:8080->80/tcp        dockercomposegalaxyannotation_proxy_1
46abbf05376c        gmod/apollo:latest                         "catalina.sh run"        9 minutes ago       Exited (137) 8 minutes ago                                        dockercomposegalaxyannotation_apollo_1
2b550adc963f        erasche/tripal:latest                      "/entrypoint.sh"         9 minutes ago       Exited (2) 8 minutes ago                                          dockercomposegalaxyannotation_tripal_1
9fa1c3a7f949        erasche/chado:latest                       "/docker-entrypoint.s"   9 minutes ago       Exited (1) 9 minutes ago                                          dockercomposegalaxyannotation_chado_1
e1ec2a9b370a        quay.io/erasche/docker-galaxy-annotation   "/usr/bin/startup"       9 minutes ago       Up 9 minutes                 21/tcp, 80/tcp, 8800/tcp, 9002/tcp   dockercomposegalaxyannotation_galaxy_1
dd7ad5371a65        postgres                                   "/docker-entrypoint.s"   9 minutes ago       Exited (1) 9 minutes ago                                          dockercomposegalaxyannotation_apollo_db_1
```

**Comment by hexylena** [comment]
I believe this is due to the presence of the tripal image which I've removed upstream because of issues.

Try cloning https://github.com/erasche/docker-compose-galaxy-annotation and see if that fixes it for you. If it does, I'll merge my upstream to GMOD here.

**Comment by juancresc** [comment]
@erasche I've cloned the repo and this is the output:

```
n# docker-compose up -d
Creating dockercomposegalaxyannotation_apollo_db_1
Creating dockercomposegalaxyannotation_chado_1
Creating dockercomposegalaxyannotation_galaxy_1
Creating dockercomposegalaxyannotation_apollo_1
Creating dockercomposegalaxyannotation_proxy_1
root@ubuntu-512mb-sfo1-01:~/docker-compose-galaxy-annotation# docker-compose ps proxy
                Name                          Command          State    Ports 
-----------------------------------------------------------------------------
dockercomposegalaxyannotation_proxy_1   nginx -g daemon off;   Exit 1      
```

**Comment by hexylena** [comment]
@juanmas07 would you mind running `docker-compose logs proxy` and paste some of the output here?

**Comment by juancresc** [comment]
$ docker-compose logs proxy
Attaching to dockercomposegalaxyannotation1_proxy_1

I had problems when downloading from the repo you sent to me. I'll try again

**Comment by hexylena** [comment]
@juanmas07 ok, would have expected the proxy (nginx) to spew error logs if it crashed with an `Exit 1`

**Comment by nathandunn** [comment]
@erasche   I'm getting the same error as @juanmas07 . I'll see if I can replicate off of your version.

**Comment by nathandunn** [comment]
This should be fixed based on #4.  

```
docker-compose-galaxy-annotation% docker-compose ps proxy
                Name                          Command          State               Ports              
-----------------------------------------------------------------------------------------------------
dockercomposegalaxyannotation_proxy_1   nginx -g daemon off;   Up      443/tcp, 0.0.0.0:32770->80/tcp 
```
